Sponsored postman fees for automatic claiming (#1080)
Consensys/doc.linea: docs/get-started/how-to/bridge.mdx

### Execution fees

:::tip Free postman fees for L1 -> L2

The postman fee for L1 -> L2 transfers (i.e. deposits to Linea) is now sponsored for all users. 

> This means that automatic claiming transfers from L1 to L2 now have no fees, apart from some cases

Reviewer: ```suggestion
This means that automatic claiming transfers from L1 to L2 now have no fees, apart from a few cases
```

  - `gas estimated` = 100,000
  - `gas limit surplus` = 6000
  - `margin` = 2
  

>   The postman fee used to be applied to automatic claiming transfers from L1 to L2, but is now free.

Reviewer: ```suggestion
  Previously the postman fee was applied to automatic claiming transfers from L1 to L2, but is now free.
```
